crd-generator/README.md

```diff
@@ -11,8 +11,8 @@ Please give it a try and report any issue you might find.
 - Several annotations are available to override the default behavior if it doesn't fit your needs
 - Validation schema is automatically derived from your class' spec implementation (or from the CR
   class, if no spec is present)
-- CRDs are automatically generated at compile-time if you include the `io.fabric8:crd-generator`
-  dependency to your project (with the suggested `compile` scope)
+- CRDs are automatically generated at compile-time if you include `io.fabric8:crd-generator-api` 
+  and `io.fabric8:crd-generator-apt` dependencies to your project (with the suggested `compile` scope)
```

Comment:
You don't need to include `crd-generator-api` if you use `crd-generator-apt`: that last one should be sufficient. The API module is only needed for programmatic access to the CRD generation.